src/components/atoms/Plugin/index.tsx

```diff
@@ -1,4 +1,5 @@
 import { forwardRef, ForwardRefRenderFunction, IframeHTMLAttributes, ReactNode } from "react";
+import type { RefObject } from "react";
```

Comment:
Can you export the RefObject from Plugin/hooks.ts and then import it with useHook?

Reply:
Sorry can you explane more about this? Since `RefObject` is a type definition from react i think i can import the type and export it from Plugin/hooks but how to import the type with useHook?

Reply:
After some more thought I'll just say leave it as is. But just so you know, I meant to do something like
```
import type { RefObject as RefObjectType } from "react";

export type RefObject = RefObjectType
```
inside the hooks.ts. Then inside the index.tsx file:
``` 
import useHooks, { RefObject } from "./hooks.ts
```
But again, since it is a react import (and not something we define ourselves), maybe just leave it as is.